doc/reference/views.rst

```diff
@@ -1296,6 +1296,10 @@ There are 5 possible type of tags in a dashboard view:
         A widget to format the value (like the widget attribute for fields).
         For example, monetary.
 
+    - ``measure`` (optional)
+        The measure to use when clicking on this aggregate, by default it uses the same
```

Comment:
you should explain more clearly that it is a string, and it should be a valid field name or __count__